### ISSUE TITLE
[Accessibility] A11y_.Net Editor: The "iFrame" in the page doesn't have a "Title" attribute which describes the iframe

### ISSUE THREAD

**Issue by ChetanDumbala** [dotnet/try#119]
**Environment Details**
https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
Browser: Edge (41.16299.402.0)
Chrome (67.0.3396.87)

**Additional Details:**
MAS Violated: MAS 2.4.2

**Repro steps:**
Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
Tab till “Enter focus mode” and press enter
Verify whether the "iFrame" in the page has a title attribute or not.

**Actual Result:**
The "iFrame" in the page doesn't have a "Title" attribute which describes the iframe

**Expected Result:**
The "iFrame" in the page is expected to have a "Title" attribute which describes the iframe

**User Impact:**
Screen reader users would face difficulty in understanding what that particular iframe contains.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={c74ff881-82b6-4401-92cf-8919f9bffaf2}

![the iframe in the page doesn t have a title attribute which describes the iframe](https://user-images.githubusercontent.com/41179360/42674301-c45cf3d4-868c-11e8-9383-155e45b02a07.JPG)

**Comment by LadyNaggaga** [comment]
@jonsequitur can  please have a look at this accessibility bug?

**Comment by jonsequitur** [comment]
@ChetanDumbala This should be filed in https://github.com/dotnet/docs, as they own the HTML for the `iframe` itself.

**Comment by LadyNaggaga** [comment]
@dend  @jdanyow could you please have a look at this accessibility bug?

**Comment by dend** [comment]
Tracking this in the docs feedback repo! 👍 Thanks folks!

**Comment by LadyNaggaga** [comment]
1.	Importance of experience (High or Low Impact). 
Impact: Low we haven't received any customer complaints directly to the service. The .NET online editor is the service behind editor and we do not own or control the UI experience. 
Describe scenario: Please the description above
2.	Quality of Experience (High or Low Impact). 
Impact: Low we haven't received any customer complaints directly to the service. The .NET online editor is the service behind editor and we do not own or control the UI experience. 
3.	[Link to AAT ](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/93711e75-9cb7-4061-b1e9-53177a087cae)
4.	Any alternate solutions or technologies evaluated
Not applicable 
5.	Ensure all evidence for hand-off to external team is provided in the bug
Issue has been resolved by the docs teams
https://github.com/MicrosoftDocs/feedback/issues/734

**Comment by umasarath52** [comment]
This was discussed with Ryan and approved to close as this is non blocking scenario.
